bhclub.js

```diff
@@ -32,3 +32,27 @@ $(document).on('click', 'main', function() {
     $('#toggle').addClass('fa-bars');
     $('main').removeClass('main-pointer');
 });
+
+// FAQ Cards hide/show
+let card = document.getElementsByClassName("card");
+for (let i = 0; i < card.length; i++) {
+    let accordion = card[i].getElementsByClassName("card-header")[0];
+    // Click should only work on accordion-header of each card
+    accordion.addEventListener("click", function() {
+
```

Comment:
nit: extra newline? not sure about code style for this; it feels unnecessary because the block is already denoted by indentations, so a blank line isn't needed

```suggestion
```